packages/cli/source/package.json

```diff
@@ -16,7 +16,7 @@
   "dependencies": {
     "@coveo/cli-commons": "1.36.0",
     "@coveo/push-api-client": "2.5.15",
-    "@coveord/platform-client": "34.13.0",
+    "@coveord/platform-client": "34.14.0",
```

Comment:
Watch out: With the current scope of the PR (feat) + this change, your merge commit would wrongly bump the minor of the package. (bump of deps alone are just patch)

The easiest way for your changes to not do that would be to have the dep bump on another PR with the chore or fix scope.

(This apply to all the other packages 👇 )